assets/src/edit-story/utils/generatePatternStyles.js

```diff
@@ -118,6 +122,9 @@ function generatePatternStyles(pattern = null, property = 'background') {
   }
 
   if (type === 'solid') {
+    if (pattern === MULTIPLE_VALUE) {
```

Comment:
Should this be outside the above `if` clause?

Reply:
Yeah, it could. The other cases throw errors because of invalid states, so yeah I think it would be fine moved up.